src/views/projects/CloneButton.svelte

```diff
@@ -1,14 +1,17 @@
 <script lang="ts">
-  import * as utils from "@app/lib/utils";
   import Clipboard from "@app/components/Clipboard.svelte";
   import Floating from "@app/components/Floating.svelte";
   import { closeFocused } from "@app/components/Floating.svelte";
 
   export let seedHost: string;
   export let id: string;
+  export let name: string;
 
-  $: radCloneUrl = `rad clone rad://${seedHost}/${utils.parseRadicleId(id)}`;
-  $: gitCloneUrl = `https://${seedHost}/${utils.parseRadicleId(id)}.git`;
+  $: radCloneUrl = `rad clone ${id}`;
+  $: gitCloneUrl = `git clone https://${seedHost}/${id.replace(
```

Comment:
If we would have a `parseRepositoryId` as mentioned in another comment, we could make use of it here instead of relying on `String.replace()`

Reply:
6e3b46c5bbe741367c826db1dc4cf31232598345